ext/CreateParametersExt.jl

```diff
@@ -0,0 +1,41 @@
+module CreateParametersExt
+
+import Thermodynamics.Parameters.ThermodynamicsParameters
+import CLIMAParameters as CP
+
+ThermodynamicsParameters(FT) = ThermodynamicsParameters(CP.create_toml_dict(FT))
+
+function ThermodynamicsParameters(toml_dict::CP.AbstractTOMLDict)
+    name_map = Dict{String, String}(
```

Comment:
Out of curiosity, why do we use strings instead of symbols? It seems that strings include names that should be invalid, whereas symbols are a better map to the concept of a "variable name". This would make using named tuples more natural as well rather than dictionaries (in this case, it seems we should use `NamedTuple` rather than `Dict` since we don't instead to add to or change the contents of `name_map`).

Reply:
Both strings and symbols are currently supported! I am happy to turn the name map into a NamedTuple, but I will have to add support for it.

Reply:
I like `NamedTuple` because it seems a little more explicit / purpose-built for this (using `Dict` implies we need to manipulate its elements, `pop!`ing, adding, etc). `NamedTuple` evokes a clearer vision of the code's purpose, making it easier to understand. Not every optimization is worth the effort, so it's up to you.

Reply:
Thanks for reviewing! I think NamedTuple is more intuitive for people following this pattern, so I will used NamedTuples as the default name map examples.